lookup_plugins/retrieve_conjur_variable.py

```diff
@@ -135,6 +142,12 @@ def run(self, terms, variables=None, **kwargs):
         if not conf:
             raise Exception('Conjur configuration should be in environment variables or in one of the following paths: \'~/.conjurrc\', \'/etc/conjur.conf\'')
 
+	# Load Conjur version
+	if (environ.get('CONJUR_VERSION') is not None):
+	    conf['version'] = environ.get('CONJUR_VERSION')
+	else:
+	    conf['version'] = "4"
```

Comment:
How about flipping this to `"5"` so we can override with 4?  The v5 appliance should be released in Q2.

Reply:
Well, you are absolutely right. To be honest did it on the rush during the PoC and moving the default to v5 is a best option.